__tests__/driver.test.js

```diff
@@ -1,7 +1,6 @@
-'use strict';
-
```

Comment:
So it looks like on class-19 of your CAPS application the package.json got deleted out of the root level so it doesn’t recognize where to run tests.